base/interactiveutil.jl

```diff
@@ -251,7 +251,8 @@ function versioninfo(io::IO=STDOUT, verbose::Bool=false)
         println(io, "DEBUG build")
     end
     println(io,             "Platform Info:")
-    println(io,             "  System: ", Sys.KERNEL, " (", Sys.MACHINE, ")")
+    println(io,             "  OS: ", is_windows() ? "Windows" : is_apple() ?
```

Comment:
See [`GetVersion`](https://msdn.microsoft.com/en-us/library/windows/desktop/ms724439(v=vs.85).aspx) on Windows, though the documention makes this sound iffy (it depends on how Julia is built); we might use the [`systeminfo` command](https://www.windows-commandline.com/find-windows-os-version-from-command/) instead?   On MacOS, it looks like ```readchomp(`sw_vers -productVersion`)``` should be reliable?